utils/json2xml.py

```diff
@@ -41,20 +41,42 @@ def generate_coco_from_via(via_annotation, prefix=""):
         coco['size'] = {"width": "300", "height":"300", "depth":"3"}
         coco["segmented"] = 0
         coco["object"] = []
+        polygon_flag='False'
         for region in image_data["regions"]:
```

Comment:
This is making all the regions for the image to be `polygon` if even one region is `polygon`, which will fail for bounding box regions. 

polygon / bbox check will be done for each region. Also, please separate the code only for lines different in input shape type as 
```suggestion
        for region in image_data["regions"]:
            details = {}
            details["name'] = ...
            ....
            if "all_points_x" in region["shape_attributes"]:
                details["truncated"] = ....
                details["bndbox'] = {...}
            else:
                details["truncated"] = ....
                details["bndbox'] = {...}
            
            coco["object"].append(details)
```